tests/vcf_io/test_vcf_io.h

```diff
@@ -161,12 +161,6 @@ SEQAN_DEFINE_TEST(test_vcf_io_read_vcf_record)
     SEQAN_ASSERT_EQ(records[2].format, "GT:GQ:DP:HQ");
     SEQAN_ASSERT_EQ(length(records[2].genotypeInfos), 3u);
 
-    for (unsigned i = 0; i < 25; ++i)
-    {
-        SEQAN_TEST_EXCEPTION(seqan::ParseError,
-                             seqan::readRecord(record, vcfIOContext, iter, seqan::Vcf()));
-        seqan::skipLine(iter);
-    }
```

Comment:
Why is was this removed?

Reply:
I put it back. The problem was there were records in the test file which were invalid for vcf format v4.1 but now they are valid per the new (v4.2) specification. I removed those records from the test file.